osf_tests/test_queued_mail.py

```diff
@@ -183,3 +185,15 @@ def test_remind_prereg_presend_deleted_draft(self, mock_mail, user, prereg):
         prereg.deleted = timezone.now()
         prereg.save()
         assert not mail.send_mail()
+
+    def test_disabled_queued_emails_not_sent_if_switch_active(self, user):
+        SwitchFactory(name=DISABLE_ENGAGEMENT_EMAILS, active=True)
```

Comment:
Nit: I think `waffle.testutils.override_switch` is preferred over `SwitchFactory`. See the CSRF tests for an example.

Reply:
Neat 👍